10981.0.txt

```diff
@@ -0,0 +1,38 @@
+@0
+
+
+
+@1
+MOV UP, DOWN		#take input from above and move it down
+
+
+@2
+
+
+
+@3
+
+
+
+@4
+ADD UP
+ADD ACC
+MOV ACC, RIGHT
+SUB ACC	#add input from above to the ACC, then add ACC to istelf, the output ACC to the right, then subtract ACC with itself 
```

Comment:
Comments should explain "why", not "how". Not clear why you're subtracting `ACC` from itself here -- doesn't that do the same as `MOV 0, ACC`?